doc/release-notes/10889_bump_PG17_FlyWay10.md

```diff
@@ -0,0 +1,3 @@
+This release bumps both the Postgres JDBC driver and Flyway versions. This should better support Postgres version 17, and as of version 10 Flyway no longer requires a paid subscription to support older versions of Postgres.
+
```

Comment:
In #9927 in the release note we said, "Postgres 13 remains the version used with automated testing."

Should we say something like that in the release note here? 🤔

Reply:
added =)